### DISCUSSION TITLE
[with-storybook] Add CSS Modules Support

### DISCUSSION THREAD

**Discussion by matamatanot** [vercel/next.js#23948] [Ideas]
# Feature request

Add configs for CSS Modules.

## Is your feature request related to a problem? Please describe.

#16139 This PR shows usage in Storybook V6 (TypeScript build-in support) and I think it's so helpful.
But, it would be more useful to add the use of CSS Modules. Most developers using Storybook apply styles to components. And, Next.js also recommends [the use of CSS Modules #15542](https://github.com/vercel/next.js/issues/15542).

## Describe the solution you'd like

Extend Webpack config.
https://github.com/vercel/next.js/blob/feeb941dc6fd44887216b0f0d257aab6bbd2625d/packages/next-plugin-storybook/preset.js#L32

## Describe alternatives you've considered
Install dependencies `style-loader css-loader sass-loader`

## Additional context

#15195
#15196
#15351
https://github.com/storybookjs/storybook/issues/9610

**Comment by hades200082** [comment]
For anyone else that finds this page and is struggling to get CSS modules working with storybook and nextjs - this package solved it for me https://github.com/Negan1911/storybook-css-modules-preset
